samples/bluetooth/peripheral/prj.conf

```diff
@@ -22,4 +22,4 @@ CONFIG_FLASH_PAGE_LAYOUT=y
 CONFIG_FLASH_MAP=y
 CONFIG_FCB=y
 CONFIG_SETTINGS=y
-CONFIG_SETTINGS_FCB=y
+CONFIG_SETTINGS_FCB=y
```

Comment:
Where did this change come from?

Reply:
Looks like an accident from the force-push 1 minute ago.

Reply:
I add CONFIG_BT_REMOTE_VERSION to the sample and removed slightly too much, should be fixed now.